objects/rct2tt/ride/rct2.tt.microbus.json

```
@@ -133,6 +133,7 @@
             "sv-SE": "Hippiebuss",
             "ko-KR": "마이크로버스 라이드",
             "zh-TW": "小型巴士遊樂設施",
+            "ja-JP": "マイクロバスモーションシム",
```

```suggestion
            "ja-JP": "マイクロバスライド",
```

「モーションシム」means motion simulator. This isn't wrong, of course, but not true to the original English string. I'd prefer to go with the shorter 「ライド」(ride)